dotCMS/src/main/java/com/dotmarketing/common/db/ForeignKey.java

```diff
@@ -0,0 +1,95 @@
+package com.dotmarketing.common.db;
+
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class ForeignKey {
+
+    private final String primaryKeyTableName;
+    private final Set<String> primaryKeyColumnNames = new LinkedHashSet<>();
+    private final String foreignKeyTableName;
+    private final Set<String> foreignKeyColumnNames = new LinkedHashSet<>();
+    private final String foreignKeyName;
+
+    public ForeignKey(String primaryKeyTableName, String foreignKeyTableName, String foreignKeyName) {
+        this.primaryKeyTableName = primaryKeyTableName;
+        this.foreignKeyTableName = foreignKeyTableName;
+        this.foreignKeyName = foreignKeyName;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj instanceof ForeignKey) {
+            ForeignKey k=(ForeignKey)obj;
+            if (!k.primaryKeyTableName.equalsIgnoreCase(primaryKeyTableName)) {
+                return false;
+            }
+            if (!k.foreignKeyTableName.equalsIgnoreCase(foreignKeyTableName)) {
+                return false;
+            }
+            if (!k.foreignKeyName.equalsIgnoreCase(foreignKeyName)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1126)